examples/vtune_integration/basic_usage_vtune_profiler.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+from numba import njit
+import sdc
+import sdc.vtune_integration as vt
+import ctypes
+import itt
+
+handle = vt.ctypes_string_handle_create(b"Head\0")
+
+functype_task_begin = ctypes.CFUNCTYPE(None, ctypes.c_void_p, ctypes.c_void_p)
+ctypes_task_begin = functype_task_begin(itt.__itt_task_begin)
```

Comment:
I believe it should be done in `vtune.py`, not here. And here just import these functions